samples/grids/data-grid/load-save-layout/src/index.tsx

```diff
@@ -0,0 +1,378 @@
+                    horizontalAlignment="stretch" width="130" paddingTop="5" paddingBottom="5" />
+
+                    <IgrTemplateColumn field="Address" headerText="Address" horizontalAlignment="left"
+                        cellUpdating={this.onAddressCellUpdating} width="*>160" />
+
+                    <IgrTemplateColumn field="Phone" horizontalAlignment="center"
+                        cellUpdating={this.onPhoneCellUpdating} width="140" />
+
+                    {/* <IgrTemplateColumn field="Email" horizontalAlignment="center"
+                    cellUpdating={this.onEmailCellUpdating} width="160" /> */}
+
+                    {/* <IgrTextColumn field="Country" width="*>140" horizontalAlignment="center"/> */}
+                    <IgrTextColumn field="Income" width="*>130" horizontalAlignment="center"/>
+                    <IgrTextColumn field="Age" width="*>110" horizontalAlignment="center"/>
+               </IgrDataGrid>
+            </div>
+        );
+    }
+
+    public onGridRef(grid: IgrDataGrid) {
+        if (!grid) { return; }
+
+        this.grid = grid;
+        this.grid.actualDataSource.isSectionExpandedDefault = true;
+        
+        if (this.toolbar !== null) {
+            this.toolbar.targetGrid = this.grid;
+        }
+    }
+
+    public onToolbarRef(toolbar: IgrDataGridToolbar) {
+          this.toolbar = toolbar;
+          if (this.toolbar !== null) {
+              this.toolbar.targetGrid = this.grid;
+          }
+    }
+
+    public componentDidMount() {
+
+        const peopleGroup = new IgrColumnGroupDescription();
+        peopleGroup.field = "Country";
+        peopleGroup.displayName = "Country";
+        this.grid.groupDescriptions.add(peopleGroup);
+
+        const incomeGroup = new IgrColumnGroupDescription();
+        incomeGroup.field = "Income";
+        incomeGroup.displayName = "Income";
+        this.grid.groupDescriptions.add(incomeGroup);
+
+        const peopleCount = new IgrColumnSummaryDescription();
+        peopleCount.field = "Photo";
+        peopleCount.operand = SummaryOperand.Count;
+        this.grid.summaryDescriptions.add(peopleCount);
+
+        const sales = new IgrColumnSummaryDescription();
+        sales.field = "Sales";
+        sales.operand = SummaryOperand.Max;
+        sales.formatOverride = new Intl.NumberFormat('en-EN', { style: 'currency', currency: 'USD', minimumFractionDigits: 0, maximumFractionDigits: 0 });
+        this.grid.summaryDescriptions.add(sales);
+
+        const salary = new IgrColumnSummaryDescription();
+        salary.field = "Salary";
+        salary.operand = SummaryOperand.Average;
+        salary.formatOverride = new Intl.NumberFormat('en-EN', { style: 'currency', currency: 'USD', minimumFractionDigits: 0, maximumFractionDigits: 0 });
+        this.grid.summaryDescriptions.add(salary);
+    }
+
+    public getProductivityChart(props: IIgrCellTemplateProps) {
```

Comment:
remove this function

Reply:
resolved

https://github.com/IgniteUI/igniteui-react-examples/pull/101/commits/af54a47b9c02875025df7f83cf81dc8e6813eb85